### ISSUE TITLE
OP warps don't work with CP when you don't have enough CP to cover gil cost

### ISSUE THREAD

**Issue by rujind** [EdenServer/community#512]
(was asked to post this, have not encountered it myself)
People are having an issue using CP to OP warp. I personally believe it is related to not having enough CP to support the old cost. In this scenario it doesn't warp you nor take your CP.

**Comment by arreisGit** [comment]
I'm being able to replicate this problem. I called a gm yesterday about it but unfortunately I can't recall its name, however,  he mentioned something about not having conquest points equal or greater to the quantity I'd have to pay with gil.

It only happens  when I try to outpost teleport with conquest points and I already saw this with two of my characters. 

Here is what I'm doing:

Nation: Windurst
Character Name: Zerreitug
Gil: 2344
Conquest Points: 186
1- Try to outpost teleport from Port Windurst to Ronfare Region
2- The NPC displays a message saying the region is under my nation's control, but it is not.
([see screenshot](http://prntscr.com/m85pib))
3-  Since the region is not under Windurst control, the price to pay is either 800 gil or 80 conquest points.
4- Select pay with Conquest points
5- The NPC will display the "teleporting message" and animation OK.
6- The character will remain invisible but no teleport will actually happen.
([see screenshot](http://prntscr.com/m85sed))
7- No conquest point was charged, and I can move my character freely.
8- If I change zone my character becomes visible again.
